### DISCUSSION TITLE
Debugging Coldtype

### DISCUSSION THREAD

**Discussion by bensapirstein** [coldtype/coldtype#137] [Q&A]
How could I debug Coldtype or have access to the interpreter during the run?

**Comment by stenson** [comment]
A great question that I don't know the answer to — I can't say I've ever been in the habit of using the debugger in python, not that it shouldn't or can't be done, just something I don't know anything about. Coldtype scripts are run via `runpy.run_path` which I'd expect to have some facilities for debugging, but I'd have to look into it further.
